test/JDBC/upgrade/latest/schedule

```diff
@@ -538,3 +538,4 @@ test_conv_string_to_datetimeoffset
 test_conv_string_to_smalldatetime
 test_conv_string_to_time
 BABEL-3401
+PARTITION
```

Comment:
(Small nit): Both Here and at other places, it is always a good practice to add newline at the end of a file.